Lib/test/test_tabnanny.py

```diff
@@ -293,8 +293,8 @@ def validate_cmd(self, *args, stdout="", stderr="", partial=False):
         _, out, err = script_helper.assert_python_ok('-m', 'tabnanny', *args)
         # Note: The `splitlines()` will solve the problem of CRLF(\r) added
         # by OS Windows.
-        out = out.decode('ascii')
-        err = err.decode('ascii')
+        out = out.decode('utf-8')
```

Comment:
I suggest to use os.fsdecode() instead. subprocess uses the locale encoding, it can be different than UTF-8. Moreover, fsdecode() uses the surrogateescape error handler for undecodable bytes.